src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java

```diff
@@ -200,8 +181,10 @@ public WorkflowRun(WorkflowJob job, File dir) throws IOException {
         }
         try {
             onStartBuilding();
+            // TODO introduce API to wrap out log location
```

Comment:
Implementation from the prototype: https://github.com/oleg-nenashev/workflow-job-plugin/tree/jw-hackaton-external-logging